### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by riccardobrasca** [comment]
Since you are at it, can you fix [this](https://github.com/leanprover-community/mathlib4/pull/5337#discussion_r1240821645) comment too?

File: Mathlib/NumberTheory/Cyclotomic/PrimitiveRoots.lean [hunk 1]
```diff
@@ -432,7 +426,8 @@ theorem sub_one_norm_prime [hpri : Fact (p : ℕ).Prime] [hcyc : IsCyclotomicExt
 -- Porting note: writing `(2 : ℕ+)` was not needed (similarly everywhere).
 theorem pow_sub_one_norm_two {k : ℕ} (hζ : IsPrimitiveRoot ζ (2 ^ (k + 1)))
     [IsCyclotomicExtension {(2 : ℕ+) ^ (k + 1)} K L]
-    (hirr : Irreducible (cyclotomic (2 ^ (k + 1)) K)) : norm K (ζ ^ 2 ^ k - 1) = (-2) ^ 2 ^ k := by
+    (hirr : Irreducible (cyclotomic (2 ^ (k + 1)) K)) :
+    norm K (ζ ^ 2 ^ k - 1) = (-2 : K) ^ 2 ^ k := by
```
Comment (riccardobrasca) [inline Mathlib/NumberTheory/Cyclotomic/PrimitiveRoots.lean:430]: I am not sure I understand the point, why `(-2 : K)` is better?

**Review by riccardobrasca — APPROVED** [review]
Thanks!

bors d+

File: Mathlib/NumberTheory/Cyclotomic/PrimitiveRoots.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/NumberTheory/Cyclotomic/PrimitiveRoots.lean:430]: The previous code parsed as `(((-2) ^ 2 ^ k : ℤ) : K)`, which didn't match mathlib3

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**